.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,38 @@
+#
+# codeQL       : https://codeql.github.com/docs/codeql-overview/
+# GH-Action    : https://github.com/github/codeql-action
+# LGTM         : https://lgtm.com/#explore
+# codeql-config: https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#using-a-custom-configuration-file
+#
+name: "CodeQL"
+
+on:
+  schedule:
+    - cron: "26 5 * * 1"
```

Comment:
Any particular reason for the time?

Reply:
I wanted to be a time where we normally are not running CI and did not want to be on o'clock , half-past or quarter ...